integration/test/basic/basic_test.go

```diff
@@ -59,6 +59,12 @@ func TestChartLifecycle(t *testing.T) {
 		t.Fatalf("could not install %q %v", cr, err)
 	}
 
+	err = r.WaitForStatus(cr, "DEPLOYED")
+	if err != nil {
+		t.Fatalf("could not get release status of %q %v", cr, err)
+	}
+	l.Log("level", "debug", "message", fmt.Sprintf("%s succesfully deployed", cr))
```

Comment:
@MarcelMue FYI. I added a wait for the e2e chart being deployed, before the wait for the test chart being deployed. 

I think this makes the test more reliable. Re-running the test. If it succeeds then I'll merge.

Reply:
<3